### DISCUSSION TITLE
Is it possible to enable parallel sync?

### DISCUSSION THREAD

**Discussion by ilmanowar** [abraunegg/onedrive#2336] [Q&A] [answered]
I have many small files to sync. Therefore the internet bandwidth is no longer limiting the speed of the sync process because it takes time to select a file for sync and open the connection.
Is it possible to process 5-10 files at the same time? This will speed up the process and use more of my internet bandwidth.

Thanks for the comments.

**Comment by abraunegg** [comment, marked as answer]
@ilmanowar 
Please refer to https://github.com/abraunegg/onedrive/issues/232
